core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -456,11 +459,12 @@ object ConsumerGroupCommand {
         CommandLineUtils.checkRequiredArgs(parser, options, groupOpt)
       if (options.has(deleteOpt) && !options.has(groupOpt) && !options.has(topicOpt))
         CommandLineUtils.printUsageAndDie(parser, "Option %s either takes %s, %s, or both".format(deleteOpt, groupOpt, topicOpt))
+      if (options.has(commandConfigOpt) && !options.has(newConsumerOpt))
```

Comment:
Nitpick: it would be a little nicer to do the commandConfig check inside the existing block where we check for opts that only work with new-consumer.